recipe/bld.bat

```diff
@@ -14,14 +14,21 @@ cmake -G "NMake Makefiles" ^
       -DBUILD_SHARED:BOOL=ON ^
       -DBUILD_TESTS:BOOL=ON ^
       -DBUILD_BENCHMARKS:BOOL=OFF ^
+      -DPREFER_EXTERNAL_SNAPPY:BOOL=ON ^
+      -DPREFER_EXTERNAL_LZ4:BOOL=ON ^
+      -DPREFER_EXTERNAL_ZLIB:BOOL=ON ^
       "%SRC_DIR%"
-if errorlevel 1 exit 1
+if %ERRORLEVEL% neq 0 exit 1
 
-cmake --build .
-if errorlevel 1 exit 1
+cmake --build . --config Release
+if %ERRORLEVEL% neq 0 exit 1
 
-ctest
-if errorlevel 1 exit 1
+ctest -C release
+if %ERRORLEVEL% neq 0 exit 1
 
-cmake --build . --target install
-if errorlevel 1 exit 1
+cmake --build . --target INSTALL --config Release
+if %ERRORLEVEL% neq 0 exit 1
+
+del %LIBRARY_BIN%\msvc*.dll
+del %LIBRARY_BIN%\Microsoft.*
+move %LIBRARY_LIB%\blosc.dll %LIBRARY_BIN%\
```

Comment:
This is interesting. Would have hoped this got installed in the right place already. Might need another look to figure out what is happening (or not happening) to require this.